src/components/OrganisationSwitcher.css

```diff
@@ -17,7 +17,7 @@
   box-shadow: 0 2px 6px 5px rgba(0, 0, 0, 0.3);
   width: 80%;
   min-height: 550px;
-  padding: 40px;
+  padding: 70px;
```

Comment:
Hi Lirry, is deze wijziging nog nodig? Je hebt namelijk al een erroroverlay css.
Misschien het beste om dit terug te veranderen voor het geval het iets kapot maakt:
Je mag het terug veranderen en een nieuwe commit doen.
Voor de rest ziet het er goed uit!
groeten Tom

Reply:
Done!